packages/common/src/utils/notifications/forgot_password_notification_service.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from "@nestjs/common";
+import { ConfigService } from "@nestjs/config";
+import { User } from "@cryptify/common/src/domain/entities/user";
+import { EmailNotificationStrategy } from "./email_notification_strategy";
+import { JwtToken } from "../../domain/jwt_token";
+
+@Injectable()
+export class ForgotPasswordService {
```

Comment:
Current name doesnt make sense, here is a better one

```suggestion
export class AuthNotificationService {
```